js-core-1/week-2/lesson.md

```diff
@@ -409,6 +409,12 @@ In the node REPL, enter the following array:
 
 2. Replace 'apple' with 'raspberry' and 'fig' with 'pineapple'
 
+### Exercise (10 mins)
+1. Write a function to remove the duplicate elements of a given array and return the new length of the array.
+Sample array: [20, 20, 30, 40, 50, 50, 50]
```

Comment:
I could achieve this by slicing off the ends:
`[20, 20, 30, 40, 50, 50, 50].slice(1,5)`